src/api.js

```diff
@@ -1,17 +1,32 @@
 import express from "express";
+import getMovieRating from "./getMovieRating.js";
+import { loadMovie } from "./movies.js";
 import { getReviewsForMovieWithId } from "./reviewList.js";
 
+
 const apiRouter = express.Router();
 
 apiRouter.get("/movies/:id", (req, res) => {
 	res.send(`movie with id of ${req.params.id}`);
 });
 
+apiRouter.get("/movies/:id/rating", async (req, res) => {
+	const id = req.params.id; 
+	const movie = await loadMovie(id)
+	const rating = await getMovieRating(id, movie.data.attributes.imdbId);
+
+	if (rating.Response == "False") {
+		res.status(502).send({ rating: "API Error" });
+	} else {
+		res.status(200).send({ rating: rating.toFixed(1)});
+	}
+
```

Comment:
The closing curly brace and parenthesis are missing here. I would expect Node to immediately complain about this so there must have been some mistake allowing this to be committed.